deployments/Dockerfiles/autonomy/Dockerfile

```diff
@@ -1,34 +1,26 @@
-ARG AEA_VERSION=latest
-FROM valory/open-aea-user:${AEA_VERSION}
-ARG AUTHOR=default_author
+FROM valory/open-aea-user:latest
 ARG AEA_AGENT
```

Comment:
why the reorg?, why the removal of version and author?

Reply:
They were constants only used once, so I just replaced them with actual values where they were used.